src/transforms/babel.js

```diff
@@ -148,6 +149,11 @@ async function getBabelConfig(asset) {
     return jsxConfig;
   }
 
+  // If there is a Flow config, return that
+  if (flowConfig) {
```

Comment:
I guess we also need to merge the flow config with the generated env/jsx configs above right?

Reply:
Probably, fixed it